web/src/routes/(user)/people/+page.svelte

```diff
@@ -463,7 +461,7 @@
             name="name"
             type="text"
             bind:value={personName}
-            bind:this={changeNameInputEl}
```

Comment:
This seems unrelated to this change and imo is not something we should encourage by making generic. I'm not sure why we need an arbitrary 100ms delay here, but there's likely something else going on that needs to be resolved.

Reply:
IIRC, it's because the FullScreenModal has a transition delay. 

I didn't want to open a PR for such a small change, but if you prefer I can open a new one.